pkg/apis/v1alpha1/gcenodeclass.go

```diff
@@ -76,6 +76,23 @@ type GCENodeClassSpec struct {
 	// +kubebuilder:validation:XValidation:message="network tag must match ^[a-z]([-a-z0-9]{0,61}[a-z0-9])?$",rule="self.all(x, x.matches('^[a-z]([-a-z0-9]{0,61}[a-z0-9])?$'))"
 	// +optional
 	NetworkTags []string `json:"networkTags,omitempty"`
+	// NetworkConfig specifies network configuration for instances
+	// +optional
+	NetworkConfig *NetworkConfig `json:"networkConfig,omitempty"`
```

Comment:
Based on GCE pages:
<img width="1028" height="2099" alt="image" src="https://github.com/user-attachments/assets/ff0a6fdb-3734-4977-841c-77369b1a79a5" />

1. Put NetworkTags into NetworkConfig field
2. It could have multiple network interface, so, `EnableExternalIPAccess` and `Subnetwork` should be array list. Something like:
```
type xxx struct {
..
  NetworkConfig NetworkConfig
}

type NetworkConfig {
  Tags []string
   NetworkInterfaces []NetworkInterface
}

type NetworkInterface {
  EnableExternalIPAccess bool
  Subnetwork string
}
```